docs/UserGuide.md

```diff
@@ -59,15 +59,36 @@ If you can type fast, Tutor's pet can get your class management tasks done faste
 
 </div>
 
-### Viewing help : `help`
+## Command summary
+
+Action | Format, Examples
+--------|------------------
+**Help** | `help`
+**Add Student** | `add-student n/NAME t/TELEGRAM_HANDLE e/EMAIL [tag/TAG]…​` <br> e.g., `add-student n/John Doe t/@johndoe e/johnd@example.com tag/student`
+**List All Students** | `list`
+**Edit Student** | `edit-student INDEX [n/NAME] [t/TELEGRAM_HANDLE] [e/EMAIL] [tag/TAG]…​`<br> e.g., `edit-student 2 n/James Lee e/jameslee@example.com`
+**Find Student** | `find-student KEYWORD [MORE_KEYWORDS]`<br> e.g., `find-student James Jake`
+**Delete Student** | `delete-student INDEX`<br> e.g., `delete-student 3`
+**Clear All Students** | `clear-student`
+**Add Class** | `add-class n/CLASS_NAME`<br> e.g., `add-class n/CS2103T Tutorial T10`
+**List Students In A Class** | `list-students c/INDEX`<br> e.g., `list-students c/3`
+**Edit Class** | `edit-class INDEX n/CLASS_NAME` <br> e.g., `edit 1 n/CS2103T Tutorial T10`
+**Find Class** | `find-class KEYWORD [MORE_KEYWORDS]`<br> e.g., `find-class CS2103T`
+**Delete Class** | `delete-class INDEX`<br> e.g., `delete-class 2`
+**Clear All Classes** |`clear-class`
+**Linking Student To A Class** | ``link s/STUDENT_INDEX c/CLASS_INDEX``<br> e.g., `find Betsy link s/1 c/2`
+**Unlink Student From A Class** | `unlink s/STUDENT_INDEX c/CLASS_INDEX` <br> e.g., `unlink s/1 c/2`
+**Exit** | `exit`
+
+### Help : `help`
 
 Shows a message explaining how to access the help page.
 
 Format: `help`
 
 ### Managing Students
 
-#### Adding a student: `add-student`
```

Comment:
I think we should retain the present participle form of the verb? The CS2101 set of slides for "Writing User Guide" also seems to favour retaining the present participle form of verbs, although it does not seem explicitly mentioned.

Reply:
Issue is that the formatting will not be standardized across the table and commands. I feel that perhaps its better if we follow the actions used in the command summary?

Reply:
I think keeping the previous would be better also... The previous formatting was based on the original user guide we got for the address book... so i think it is fine that the table and commands not consistent.

Reply:
Ok lets keep the previous